extern/drawdance/generators/protogen/protocol.yaml

```diff
@@ -384,48 +410,44 @@ LayerRetitle:
 LayerOrder:
     id: 133
     comment: |
-             Reorder layers
-
-             The layer tree of the given group (0 means whole tree) will be reordered
-             according to the given order.
-             The order should describe a tree using (child count, layer ID) pairs.
+             Layer order change command (legacy, subsumed by LayerTreeOrder)
 
-             For example (indented for clarity):
+             New layers are always added to the top of the stack.
+             This command includes a list of layer IDs that define the new stacking order.
 
-               2, 1,
-                 0, 10,
-                 0, 11,
-               0, 2,
-               2, 3,
-                 1, 30,
-                   0, 31
-                 0, 32
+             An order change should list all layers in the stack, but due to synchronization issues, that
+             is not always possible.
+             The layer order should therefore be sanitized by removing all layers not in the current layer stack
+             and adding all missing layers to the end in their current relative order.
 
-              Each layer in the group must be listed exactly once in the new order,
-              or the command will be rejected.
+             For example: if the current stack is [1,2,3,4,5] and the client receives
+             a reordering command [3,4,1], the missing layers are appended: [3,4,1,2,5].
 
+             If layer controls are locked, this command requires session operator privileges.
     fields:
-      - root u16: hex
       - layers vec_u16: hex
 
 LayerDelete:
     id: 134
     name: deletelayer
     comment: |
-             Delete a layer
+             Layer deletion command (legacy, subsumed by LayerTreeDelete)
 
-             If the merge attribute is nonzero, the contents of the layer is merged
-             to the layer with the given ID.
+             If the merge attribute is set, the contents of the layer is merged
+             to the layer below it. Merging the bottom-most layer does nothing.
 
              If the current layer or layer controls in general are locked, this command
              requires session operator privileges.
     fields:
         - id u16: hex
-        - merge_to u16: hex
+        - merge bool
 
 LayerVisibility:
     id: 135
-    comment: Toggle layer local visibility (this is used internally only and never sent over the network)
+    comment: |
+             Removed in version 2.2, but still included for compatibility with
+             2.1 sessions that might send it, the client ignores them though.
```

Comment:
Because when you join a Drawpile 2.1 session, they'll still appear and are valid, so I didn't wanna spew errors about it. They simply get accepted and thrown on the floor.